.github/workflows/build-release.yaml

```diff
@@ -13,7 +13,7 @@ env:
 
 jobs:
   create-release:
-    runs-on: ubuntu-latest
```

Comment:
Maybe we should not get this in since it doesnt resolve the win-x64 bug, and switching to this shouldnt matter anyway since we run our builds entirely within the builder container.